### ISSUE TITLE
Expanded rows - concise display status

### ISSUE THREAD

**Issue by Camil88** [glin/reactable#180]
Hello, I have a question regarding current status of displaying rows in concise manner. It's best described here: #147 
It would be extremely useful not only for me I guess but for many other users as well.

**Comment by glin** [comment]
Agreed that it would be extremely useful for many users. However, I don't have any update beyond my last comment on that issue (https://github.com/glin/reactable/issues/147#issuecomment-830696597), sorry.
